benches/car-index.rs

```diff
@@ -0,0 +1,66 @@
+// Copyright 2019-2023 ChainSafe Systems
```

Comment:
Is this benchmark compiled in the CI?

Reply:
It is not. :( We'll fix that in a follow-up PR.